options.go

```diff
@@ -64,6 +64,16 @@ func AddCaller() Option {
 	})
 }
 
+// AddCallerSkip configures the Logger to annotate each message with the filename
+// and line number of a caller outside of zap; which caller in the stack is
+// controlled by the skip argument. If more than one AddCallerSkip option is
+// provided, then the maximum skip amount is used.
```

Comment:
nit: "the *sum* of the skip amounts is used." Also, to me the comment suggests that this option also enables caller annotations - may want to clarify.